category_regexes_example.csv

```diff
@@ -0,0 +1 @@
+regex;sub-category;category
```

Comment:
Keep the default categories and add a comment specifying the format instead.

The file name should still be `categories.example.csv`, correct the filename used in the code.